lisp/ess-inf.el

```diff
@@ -165,15 +165,13 @@ This may be useful for debugging."
   (run-hooks 'ess-pre-run-hook)
   (let* ((dialect (eval (cdr (assoc 'ess-dialect customize-alist))))
          (process-environment process-environment)
-         ;; Use dialect if not R, R program name otherwise
-         (temp-dialect (if ess-use-inferior-program-in-buffer-name ;VS[23-02-2013]: FIXME: this should not be here
-                           (if (string-equal dialect "R")
-                               (file-name-nondirectory inferior-ess-r-program)
-                             dialect)
-                         dialect))
-         (inf-buf (inferior-ess--get-proc-buffer-create temp-dialect))
+         (inf-buf (inferior-ess--get-proc-buffer-create
```

Comment:
Is this change related to xref?

Reply:
crap, commuted some unrelated code. Sharp eye!